tests/bud.bats

```diff
@@ -7313,6 +7313,13 @@ _EOF
   local_head_hash=$output
   run_buildah run testctr -- sh -c 'cd podman-tag && git ls-remote --tags origin v5.0.0^{} | cut -f1'
   assert "$output" = "$local_head_hash"
+
+  cat > $contextdir/Dockerfile << _EOF
+FROM scratch
+ADD https://github.com/openshift/microshift.git#foobar /src
```

Comment:
We have no direct control over this repo, if they ever add such a branch all our CI would break.
Can we use buildah instead?